src/main/java/kostka/moviecatalog/repository/MovieRepository.java

```diff
@@ -4,6 +4,9 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface MovieRepository extends JpaRepository<Movie, Long> {
+    List<Movie> findByIdIn(List<Long> ids);
```

Comment:
OT: Prý je dobré ještě požívat Set místo List. Prý to má v Hibernate někdy vliv na výkon, Ale nevím jestli i tady, nebo jen u asociací. To se musím znovu podívat :D

Ale to píš jen tak. Dávám, palec.